govc/vm/customize.go

```diff
@@ -75,6 +77,8 @@ func (cmd *customize) Register(ctx context.Context, f *flag.FlagSet) {
 	f.Var(&cmd.dnssuffix, "dns-suffix", "DNS suffix list")
 	cmd.dnssuffix = nil
 	f.StringVar(&cmd.kind, "type", "Linux", "Customization type if spec NAME is not specified (Linux|Windows)")
+	f.StringVar(&cmd.username, "username", "", "for windows vms, full name of the end user in firstname lastname format")
```

Comment:
By convention, flag help text starts w/ upper-case 'For ...', how about prefix these both with:
> Windows only: full name of ...

Reply:
Thank you for the review and suggestion. I have updated the description and the go doc accordingly.